dev/RadioButtons/RadioButtons.cpp

```diff
@@ -196,6 +198,27 @@ void RadioButtons::OnChildPreviewKeyDown(const winrt::IInspectable&, const winrt
     }
 }
 
+void RadioButtons::OnAccessKeyInvoked(const winrt::UIElement&, const winrt::AccessKeyInvokedEventArgs& args)
+{
+    if (m_selectedIndex)
+    {
+        if (auto const repeater = m_repeater.get())
+        {
+            if (auto const selectedItem = repeater.TryGetElement(m_selectedIndex))
+            {
+                if (auto const selectedItemAsControl = selectedItem.try_as<winrt::Control>())
+                {
+                    selectedItemAsControl.Focus(winrt::FocusState::Keyboard);
+                    return;
+                }
+            }
+        }
+    }
+    // If we don't have a selected index, focus the RadioButton's which under normal
+    // circumstances will put focus on the first radio button.
+    this->Focus(winrt::FocusState::Keyboard);
```

Comment:
Do we need to mark args.Handled if we do set focus or should it still keep bubbling up.

Reply:
If the args are handled, we will not receive the event (you have to register with the handledToo flag set to true to see handled events).  Maybe we should mark it as handled though.. I'm not so familiar with access keys, do you know who I could ask?

Reply:
AccessKeyInvoked is not a routed event, so it can still arrive to you if args.Handled is true I believe - if an earlier handler set it to handled before you.  In this case, you subscribe in the constructor before anybody else could, so it's ok, but in general be aware that "Handled" doesn't guarantee it's not true by the time you receive it - only RoutedEvents do that.